app/views/visitors/index.html.erb

```diff
@@ -1 +1,3 @@
-<h3>Welcome</h3>
+<%= content_for :page_title_prefix, I18n.t('page_title.home') %>
+
+<h3>BEIS - Report Overseas Development Assistance</h3>
```

Comment:
We could use our new local file to supply this content consistently across the app with `I18n.t(app.title)`.